deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset-configmap.yaml

```diff
@@ -480,6 +483,7 @@ data:
               - socket_summary
             system.hostfs: /hostfs
       - name: kubernetes-node-metrics
+        id: kubernetes-node-metrics
```

Comment:
I believe agent ignores the `name` key above, I suspect we could just change the `name` to `id` instead of duplicating it but want to double check the name isn't important when running under k8s.

Reply:
Yeap, I think the name is not important here. At least for cnative team this is not sth we specificaly handle, so if it's ok for Agent then we can change it.

Reply:
Thanks, I have replaced `name: ` with `id: ` everywhere.